core/src/main/java/org/apache/gravitino/EntityStore.java

```diff
@@ -213,6 +214,15 @@ default SupportsTagOperations tagOperations() {
     throw new UnsupportedOperationException("tag operations are not supported");
   }
 
+  /**
+   * Get the extra policy operations that are supported by the entity store.
+   *
+   * @return the policy operations object that are supported by the entity store
+   */
+  default SupportsPolicyOperations policyOperations() {
+    throw new UnsupportedOperationException("policy operations are not supported");
+  }
```

Comment:
Please use the `relationOperations`, don't follow the tag-related operations, it is the legacy code.

Reply:
fixed